### PR TITLE
Added multicore support for Trim Galore!

### PR DESCRIPTION
**Pull request by drewjbeh**
Many thanks to contributing to nf-core/chipseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] `CHANGELOG.md` is updated

**Learn more about contributing:** https://github.com/nf-core/chipseq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### `Added`
 
+* Added multicore support for Trim Galore!
```
Comment (drpatelh) [inline CHANGELOG.md]:
```suggestion
* [#63](https://github.com/nf-core/atacseq/issues/63) - Added multicore support for Trim Galore!
```

Comment (drpatelh) [reply CHANGELOG.md]: Be good to link directly to the issue even if it's in a different repo :+1:

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +39,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Update phantompeakqualtools `1.2` -> `1.2.1.1`
 * Update bioconductor-deseq2 `1.20.0` -> `1.26.0`
 * Update bioconductor-vsn `3.46.0` -> `3.54.0`
+* Added pigz `2.3.4`
```
Comment (drpatelh) [inline CHANGELOG.md]:
```suggestion
* Add pigz `2.3.4`
```

**Comment by drpatelh** [comment]
Man of your word @drewjbeh :sunglasses: Thanks a bunch!

**Comment by drewjbeh** [comment]
> Man of your word @drewjbeh 😎 Thanks a bunch!
No worries! Thanks for your help - learning a lot over here. 

I'm happy to help out where I can as a bit of a side project and for some experience and leaning. Keep me in mind in the future :)

**Comment by drpatelh** [comment]
No worries. Will do :+1:

**Pull request merged**